resources/import.py

```diff
@@ -5,25 +5,25 @@
 import os
 from common import *
 
+# for import, we assign the s3 credentials to an environment variable (vs with export, we use the bound service)
+# this allows for us to use a s3 bucket in another environment by just putting the credentials in the env vars.
```

Comment:
:thinking: Perhaps not now, but would it make sense to offer this flexibility with the export too?